views/admin/javascripts/exhibits.js

```diff
@@ -2,6 +2,63 @@ var Omeka = Omeka || {};
 Omeka.ExhibitBuilder = {};
 
 (function ($) {
+    Omeka.ExhibitBuilder.deleteElement = function (element, event) {
+        event.preventDefault();
+        $(element).toggleClass('undo-delete')
+            .parent().toggleClass('deleted')
+            .siblings('div').toggleClass('frozen');
+
+        var target = $(element).parent().parent();
+        var removedClass = 'removed';
+        if (!target.hasClass(removedClass)) {
+            target.addClass(removedClass);
+            target.find('input, select, textarea').prop('disabled', true);
+        } else {
+            target.removeClass(removedClass);
+            target.find('input, select, textarea').each(function () {
+                if (!$(element).parent().parent().hasClass(removedClass)) {
+                    element.disabled = false;
+                }
+            });
+        }
+    }
+    Omeka.ExhibitBuilder.setSearchVisibility = function(show) {
+        var searchForm = $('#page-search-form');
+        var searchButton = $('#show-or-hide-search');
+
+        if (typeof show === 'undefined') {
+            show = !searchForm.is(':visible');
+        }
+        if (show) {
+            searchForm.show();
+            searchButton.addClass('hide-form').removeClass('show-form');
+        } else {
+            searchForm.hide();
+            searchButton.addClass('show-form').removeClass('hide-form');
+        }
+    }
+
+    Omeka.ExhibitBuilder.loadItemOptionsForm = function(data, itemOptionsUrl, panel, options, tinyMceCaption) {
```

Comment:
Woops, I had intended to pass that parameter to the call to `tinymce.get()` on line 53, e.g.

```
   Omeka.ExhibitBuilder.loadItemOptionsForm = function(data, itemOptionsUrl, panel, options, tinyMceCaption) {
        $(panel).addClass('loading');
        $.ajax({
            url: itemOptionsUrl,
            method: 'POST',
            dataType: 'html',
            data: data,
            success: function (response) {
                if (typeof data.caption !== 'undefined') {
                    if (!data.caption) {
                        data.caption = '';
                    }
                    tinymce.get(tinyMceCaption).setContent(data.caption);
                }
                $(options).html(response);
            },
            complete: function() {
                $(panel).removeClass('loading');
            }
        });
    };
```

It might make sense just to leave it out and make the call to tinymce.get accept a hard coded parameter (`'attachment-caption'`) since I don't think we're ever going to want to pass anything other than that there.